packages/mobile/src/config.ts

```diff
@@ -163,3 +163,5 @@ export const OTA_TRANSLATIONS_FILEPATH = `file://${CachesDirectoryPath}/translat
 export const FETCH_TIMEOUT_DURATION = 15000 // 15 seconds
 
 export const DEFAULT_APP_LANGUAGE = 'en-US'
+
+export const DEFAULT_SENTRY_TRACES_SAMPLE_RATE = 0.2
```

Comment:
The docs recommended choosing a lower number if unsure of what to use. 0 = no events are sent, 1 = 100% events are sent. happy to take suggestions, i thought 0.2 might be a nice starting point.